projects/addon-editor/components/editor/editor.component.less

```diff
@@ -5,12 +5,15 @@
     display: block;
     font: var(--tui-font-text-m);
     border-radius: var(--tui-radius-m);
+    max-height: inherit;
+    min-height: 10rem;
```

Comment:
Is it expected ?
![editor-groups](https://github.com/Tinkoff/taiga-ui/assets/35179038/cc4a297d-20a4-413d-8d62-78796feaf1fe)

Reply:
Yes, because now we do correct inherit max-height

<img width="475" alt="image" src="https://github.com/Tinkoff/taiga-ui/assets/12021443/d89e691a-3e38-437e-95ed-02b4bc51f393">